library/src/window_impl.cxx

```diff
@@ -122,6 +115,20 @@ window& window_impl::setSize(int width, int height)
   return *this;
 }
 
+//----------------------------------------------------------------------------
+window& window_impl::setPosition(int x, int y)
+{
+#if __APPLE__
+  // vtkCocoaRenderWindow has a different behavior than other render windows
+  // https://gitlab.kitware.com/vtk/vtk/-/issues/18681
+  int* screenSize = this->Internals->RenWin->GetScreenSize();
+  this->Internals->RenWin->SetPosition(x, screenSize[1] - y);
```

Comment:
You are still positioning the bottom left corner of the window, relative to the top left of the screen.  
Should be `screenSize[1] - winSize[1] - y`. However in this case, the height of the title bar is missing...

Reply:
Ok, better then nothing.

However, in that case I'm not sure to follow your comment about the window being outside of the screen in 50,50 though.

Reply:
Right, I'm confused too. My understanding right now is the (50,50) is set before the first render, so the size is still (0,0), and top left / bottom left is the same position. Then, when expanding, the window is expanded in the bottom direction.  
It's just a guess, it needs to be clarified.

Reply:
please test the new impl